src/App.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
-import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import NavBar from './components/NavBar';
+import Home from './components/Home';
+import Quote from './components/Quote';
+
+import './App.css';
 
 function App() {
-  return <Calculator />;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<NavBar />}>
+          <Route index element={<Home />} />
+          <Route path="calculator" element={<Calculator />} />
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", kindly update the `calculator page` to match the wireframe as shown in the screenshot below. 

***The wireframe***
![Screen Shot 2022-05-03 at 02 57 41](https://user-images.githubusercontent.com/65852621/166428305-e819ce57-465c-4cc8-98ba-4cb2cd1dd609.png)
***Your implementation***
![Screen Shot 2022-05-03 at 02 56 58](https://user-images.githubusercontent.com/65852621/166428347-fa083183-6408-4810-b612-fd491e275580.png)